usagov/vote-gov-drupal
# Feature/vote 2884/state enhancement online

## Jira ticket

Insert a link to the Jira ticket (e.g VOTE-XXX).

## Description

Configure the display of the state online registration block.

## Deployment and testing

### Post-deploy steps

1. lando retune

### QA/Testing instructions

1. Login as admin and create a "State content display" block. Set the general online fields for English and Spanish. Use the content in [this document](https://docs.google.com/document/d/1TVAf-R2nLTKh7kmCTe3mBKSqWn8hcGr1TsjirdCXptQ/edit#heading=h.w121gvnnkl64).
2. View the English Alabama page and confirm the "Online" block displays with the corresponding content.
3. View the Spanish Alabama page and confirm the "Online" block displays with the corresponding content.
4. Verify the registration date correctly displays for both English and Spanish.
5. Edit the Alabama page. Uncheck the "online" registration checkbox.
6. Confirm the "online" block does not display for either language.
7. Re-enable the "online" registration type.
8. In the state page configuration, set the online override fields for both English and Spanish.
9. View the English Alabama page and confirm the "Online" block displays with the override content.
10. View the Spanish Alabama page and confirm the "Online" block displays with the override content.

## Checklist for the Developer




## Checklist for the Peer Reviewers
